www/src/components/LayoutFooter/FooterLink.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import React from 'react';
+import {colors} from 'theme';
+import ExternalLinkSvg from 'templates/components/ExternalLinkSvg';
+
+const FooterLink = ({children, target, to}) => (
+  <a
```

Comment:
This should use `gatsby-link` for internal links — probably the cause of https://github.com/facebook/react/issues/10898

Reply:
Cool. I suspected that was the case but hadn't had time to dig in yet. Thanks.